scripts/scr_management/scr_management.gml

```diff
@@ -3,185 +3,340 @@ function scr_management(argument0) {
+		
+	    for (var i = 1; i <= 200; i++) {
+			if (obj_ini.role[0, i] == "Master of the Apothecarion") {
+				num[1] += 1;
+				if (nam[1] == "") then nam[1] = obj_ini.name[0, i];
+			}
+    
+			if (obj_ini.role[0, i] == obj_ini.role[100, 15]) then num[2] += 1;
+    
+			if (obj_ini.role[0, i] == string(obj_ini.role[100, 15]) + " Aspirant") then num[3] += 1;
+			// if (obj_ini.role[0, i] == "Sister Hospitaler") then num[4] += 1;
+		}
+		
 	    with(obj_managment_panel){if (manage!=obj_controller.temp[71]) then instance_deactivate_object(id);}
-	    if (num[1]>0){q+=1;obj_managment_panel.line[q]=string(nam[1]);obj_managment_panel.italic[q]=1;}
-	    if (num[2]>0){q+=1;obj_managment_panel.line[q]=string(num[2])+"x "+string(nam[2]);}
-	    if (num[3]>0){q+=1;obj_managment_panel.line[q]=string(num[3])+"x "+string(nam[3]);}
+		
+	    if (num[1]>0){
+			q+=1;obj_managment_panel.line[q]=string(nam[1]);
+			obj_managment_panel.italic[q]=1;
+		}
+	    if (num[2]>0){
+			q+=1;
+			obj_managment_panel.line[q]=string(num[2])+"x "+string(nam[2]);
+		}
+	    if (num[3]>0){
+			q+=1;
+			obj_managment_panel.line[q]=string(num[3])+"x "+string(nam[3]);
+		}
 	    // if (num[4]>0){q+=1;obj_managment_panel.line[q]=string(num[4])+"x "+string(nam[4]);}
 	    instance_activate_object(obj_managment_panel);
-    
-	    // Reclusium
-	    company=0;obj_controller.temp[71]=14;var q;q=0;
-	    i=-1;repeat(50){i+=1;num[i]=0;nam[i]="";}
-	    nam[2]=obj_ini.role[100,14];nam[3]=string(obj_ini.role[100,14])+" Aspirant";
-	    i=0;repeat(200){i+=1;
-	        if (obj_ini.role[0,i]="Master of Sanctity"){num[1]+=1;if (nam[1]="") then nam[1]=obj_ini.name[0,i];}
-	        if (obj_ini.role[0,i]=obj_ini.role[100,14]) then num[2]+=1;
-	        if (obj_ini.role[0,i]=string(obj_ini.role[100,14])+" Aspirant") then num[3]+=1;
-	    }
+	
+	    // ** Reclusium **
+		q=0;
+	    company=0;
+		obj_controller.temp[71]=14;
+		
+	    for (var i = 0; i < 50; i++) {
+			num[i] = 0;
+			nam[i] = "";
+		}
+		
+	    nam[2]=obj_ini.role[100,14];
+		
+		// Ranks
+		nam[3]=string(obj_ini.role[100,14])+" Aspirant";
+		
+	    for (var i = 1; i <= 200; i++) {
+			if (obj_ini.role[0, i] == "Master of Sanctity") {
+				num[1] += 1;
+				if (nam[1] == "") then nam[1] = obj_ini.name[0, i];
+			}
+    
+			if (obj_ini.role[0, i] == obj_ini.role[100, 14]) then num[2] += 1;
+    
+			if (obj_ini.role[0, i] == string(obj_ini.role[100, 14]) + " Aspirant") then num[3] += 1;
+		}
+		
 	    with(obj_managment_panel){if (manage!=obj_controller.temp[71]) then instance_deactivate_object(id);}
-	    if (num[1]>0){q+=1;obj_managment_panel.line[q]=string(nam[1]);obj_managment_panel.italic[q]=1;}
-	    // if (global.chapter_name!="Space Wolves") 
+		
+	    if (num[1]>0){
+			q+=1;
+			obj_managment_panel.line[q]=string(nam[1]);
+			obj_managment_panel.italic[q]=1;
+		}
+		
+		// TODO add specific Space Wolves and successor chapter logic for Master of Sanctity
```

Comment:
for a later PR?

Reply:
Yes, it would need a whole new refactor of a lot of code, since SW have a ton of special names and they handle chaplains and apothecaries differently from what i recal from lore. Same type of PR would be needed for say Iron Hands